pkg/xtrabackup/xtrabackup.go

```diff
@@ -330,6 +432,73 @@ func RestoreJob(
 	}
 }
 
+func DeleteJob(cr *apiv1alpha1.PerconaServerMySQLBackup, conf *apiv1alpha1.SidecarBackupConfig) *batchv1.Job {
```

Comment:
What do you think about naming this like `GetDeleteJob` or something like that (maybe `getDeletionJon`), because I initially thought that you want to delete a job with this method :)

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/158/commits/4528a32e0c26a104b9842b349e6439428d2c7bb7